frontend/src/components/menu/MessageIndicator.test.ts

```diff
@@ -1,29 +1,36 @@
 import { mount } from '@vue/test-utils'
-import { describe, it, expect, beforeEach, afterEach } from 'vitest'
+import { describe, it, expect, afterEach } from 'vitest'
 
 import MessageIndicator from './MessageIndicator.vue'
 
-describe('LogoImage', () => {
-  const Wrapper = () => {
-    return mount(MessageIndicator)
+describe('MessageIndicator', () => {
+  const Wrapper = (options = { props: { numberOfMessages: 1 } }) => {
+    return mount(MessageIndicator, options)
   }
 
   let wrapper: ReturnType<typeof Wrapper>
 
-  beforeEach(() => {
+  afterEach(() => {
+    wrapper.unmount()
+  })
+
+  it('renders with 1 message', () => {
     wrapper = Wrapper()
+    expect(wrapper.element).toMatchSnapshot()
   })
 
-  afterEach(() => {
-    wrapper.unmount()
+  it('renders with 2 messages', () => {
+    wrapper = Wrapper({ props: { numberOfMessages: 2 } })
+    expect(wrapper.element).toMatchSnapshot()
   })
 
-  it('renders', () => {
+  it('renders with 0 messages', () => {
+    wrapper = Wrapper({ props: { numberOfMessages: 2 } })
```

Comment:
```suggestion
    wrapper = Wrapper({ props: { numberOfMessages: 0 } })
```
according to the it statement?